### PR TITLE
[chore](cloud) log the num segments of rowset meta

### PR DESCRIPTION
**Pull request by w41ter**

### CODE REVIEW

**Comment by w41ter** [comment]
run buildall

**Review by dataroaring — DISMISSED** [review]
LGTM

**Comment by w41ter** [comment]
run buildall

**Comment by w41ter** [comment]
run buildall

**Review by dataroaring — DISMISSED** [review]
LGTM

File: cloud/src/meta-service/meta_service_txn.cpp [hunk 1]
```diff
@@ -1269,6 +1270,12 @@ void commit_txn_immediately(
                     max_size = v.ByteSizeLong();
                     max_idx = i;
                 }
+                if (v.num_segments() >= 100) {
```
Comment (gavinchou) [inline cloud/src/meta-service/meta_service_txn.cpp:1273]: record a max_num_seg  and log it at line 1274

**Comment by w41ter** [comment]
run buildall

**Review by gavinchou — APPROVED** [review]

**Review by gavinchou — APPROVED** [review]

**Review by Yukang-Lian — APPROVED** [review]

**Pull request merged**